includes/admin/pureclarity-admin.php

```diff
@@ -732,36 +602,6 @@ private function get_advanced_fields() {
 			'pureclarity_bmz_debug_callback',
 			true, // checkbox.
 		);
-		$searchSelector             = array(
-			'pureclarity_search_selector',
-			'Autocomplete Input DOM Selector',
-			'searchselector_callback',
-			false,
-		);
-		$searchResultSelector       = array(
-			'pureclarity_search_result_selector',
-			'Search Results DOM Selector',
-			'searchresults_selector_callback',
-			false,
-		);
-		$productListResultSelector  = array(
-			'pureclarity_prodlist_result_selector',
-			'Product List DOM Selector',
-			'prodlist_selector_callback',
-			false,
-		);
-		$shopEnabledCheckbox        = array(
-			'pureclarity_shop_enabled',
-			'Enable Shop',
-			'shop_enabled_callback',
-			true,
-		);
-		$shopSelector               = array(
```

Comment:
what is the shopSelector used for?

Reply:
It was a type of product listing page, the shopSelector was for a dom element that we could inject our own product listing on top of. It'd stopped being used in a previous version